README.md

```diff
@@ -74,11 +92,12 @@ i3s_client = OneviewSDK::ImageStreamer::Client.new(
   ssl_enabled: true,                  # This is the default and strongly encouraged
   logger: Logger.new(STDOUT),         # This is the default
   log_level: :info,                   # This is the default
-  api_version: 300                    # Defaults to Min supported API version, however user should change it to max supported API version (For I3S 5.40 it is 2000)
+  api_version: 2000                    # Defaults to minimum of 300 and appliance's max API version
 )
```

Comment:
Defaults to either 300 or max API version, not both. It should be appliance max version here.

Reply:
done